engines/chakra/index.js

```diff
@@ -18,5 +18,5 @@ const updateEngine = require('../../shared/engine.js');
 module.exports = updateEngine({
 	name: 'Chakra',
 	id: 'chakra',
-	verifyChecksum: true,
+	verifyChecksum: false,
```

Comment:
Apparently there should be a way to get the checksum but it seems to be broken too: https://github.com/Microsoft/ChakraCore/issues/5945#issuecomment-464809472

Reply:
Thanks @akoeplinger, that’s good to know! 

I was considering removing the checksum logic altogether, since it doesn’t really _add_ any security. But if we can keep it for now by using the `_s` resource, I could do that as part of this PR.

Reply:
Yeah I'm not sure it's worth spending much time on it. 

https://aka.ms/chakracore/cc_linux_x64_1_11_5_s for example results in a .tar.gz that contains a file which contains the sha + filename, this would probably need a bunch of custom logic for questionable value.

Reply:
That’s super weird indeed. (Why?!) I’ll just merge this as-is and then drop checksum support in a follow-on PR.